lib/dart.js

```diff
@@ -19,6 +19,11 @@ var sendLoadException = function(message) {
   }, window.location.origin);
 }
 
+// Listen for loadException events and forward to the server.
+window.addEventListener('loadException', function(e) {
```

Comment:
nit: consider something a bit more specific like `dartTestLoadException` or similar, since this is on the global `window` object you may run into conflicts

Reply:
Changed to dartLoadException given that this logic can reside outside of tests.